packages/stripe-integration/src/stripe-upe/StripeUPEPaymentMethod.spec.tsx

```diff
@@ -86,50 +96,68 @@ describe('when using StripeUPE payment', () => {
         jest.clearAllMocks();
     });
 
-    it('renders as hosted widget component', () => {
-        const container = mount(<PaymentMethodTest {...defaultProps} method={method} />);
-        const component = container.find(HostedWidgetPaymentComponent);
+    it('match a snapshot', () => {
```

Comment:
we don't use snapshots anymore

Reply:
Updated. Also coverage remained the same.